### PR TITLE
docs: update docs to capture Permissionless ICS

### PR DESCRIPTION
**Pull request by insumity**
## Description

This is a first attempt at updating the docs to capture the latest Permissionless ICS changes.
This PR does **not** yet change the docs under:
- `docs/docs/validators/changeover-procedure.md`
- `docs/docs/validators/joining-neutron.md`
- `docs/docs/validators/joining-stride.md`

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct `docs:` prefix in the PR title
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary 
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)
- [ ] Checked that the documentation website can be built and deployed successfully (run `make build-docs`)

### CODE REVIEW

File: docs/docs/adrs/adr-019-permissionless-ics.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ message MsgCreateConsumer {
 }
 ```
 
-Note that `metadata` is a required field, while the `initialization_parameterrs` and `power_shaping_parameters` are optional and can later be set using `MsgUpdateConsumer`.
+Note that `metadata` is a required field, while the `initialization_parameters` and `power_shaping_parameters` are optional and can later be set using `MsgUpdateConsumer`.
```
Comment (insumity) [inline docs/docs/adrs/adr-019-permissionless-ics.md:174]: Took into account [older comments](https://github.com/cosmos/interchain-security/pull/2261#discussion_r1757125037) that were added after the PR got merged.

File: proto/interchain_security/ccv/provider/v1/provider.proto [hunk 2]
```diff
@@ -449,11 +449,34 @@ message PowerShapingParameters {
   // have to validate the proposed consumer chain. top_N can either be 0 or any value in [50, 100].
   // A chain can join with top_N == 0 as an Opt In chain, or with top_N ∈ [50, 100] as a Top N chain.
   uint32 top_N = 1;
-  // Corresponds to the maximum power (percentage-wise) a validator can have on the consumer chain. For instance, if
-  // `validators_power_cap` is set to 32, it means that no validator can have more than 32% of the voting power on the
-  // consumer chain. Note that this might not be feasible. For example, think of a consumer chain with only
-  // 5 validators and with `validators_power_cap` set to 10%. In such a scenario, at least one validator would need
-  // to have more than 20% of the total voting power. Therefore, `validators_power_cap` operates on a best-effort basis.
+  // `validators_power_cap` corresponds to the maximum power (percentage-wise) a validator can have on the consumer chain.
```
Comment (insumity) [inline proto/interchain_security/ccv/provider/v1/provider.proto:452]: People seemed to have difficulty understanding the validators power cap, so adding more examples here to make it more accessible. (cc @ve6a)

File: docs/docs/features/power-shaping.md [hunk 3]
```diff
@@ -22,16 +22,38 @@ Note that this parameter only applies to Opt In consumer chains (i.e., with Top
 
 ### Capping the validator powers
 
-The consumer chain can specify a maximum fraction of the total voting power any of its validators should have.
-This is a security measure that makes it harder for a single large validator to take over a consumer chain: 
-It mitigates the risk of an Opt In chain with only a few validators being dominated by a validator with a large amount of stake.
-For example, setting this fraction to 33% would mean that no single validator can have more than 33% of the total voting power on the consumer, and thus no single validator would be able to stop the consumer by going offline.
+The consumer chain can specify a _power cap_ which corresponds to the maximum power (percentage-wise) a validator can have on the consumer chain.
```
Comment (insumity) [inline docs/docs/features/power-shaping.md:25]: People seemed to have difficulty understanding the validators power cap, so adding more examples here to make it more accessible. (cc @ve6a)

File: docs/docs/features/reward-distribution.md [hunk 4]
```diff
@@ -16,14 +16,63 @@ This will preserve the `ibc denom` that may already be in use.
...
-Once on the provider, the ICS rewards are distributed to the opted in validators and their delegators. 
+Once on the provider, the ICS rewards are distributed to the opted in validators and their delegators.
+Note that rewards are **only** distributed to validators that are opted in and have been validating the consumer chain
+for a continuous number of epochs (see `NumberOfEpochsToStartReceivingRewards` param).
+
 To avoid spam, the provider must whitelist denoms before accepting them as ICS rewards.  
 
+## Reward distribution with power capping
+
+If a consumer chain has set a [validators-power cap](./power-shaping#capping-the-validator-powers), then the total received
```
Comment (insumity) [inline docs/docs/features/reward-distribution.md:27]: @ve6a, I'm adding more information here on the reward distribution to hopefully make it more understandable to users.

File: docs/docs/features/reward-distribution.md [hunk 5]
```diff
@@ -16,14 +16,63 @@ This will preserve the `ibc denom` that may already be in use.
...
 For more details, see the [changeover procedure](../consumer-development/changeover-procedure.md).
 
-Once on the provider, the ICS rewards are distributed to the opted in validators and their delegators. 
+Once on the provider, the ICS rewards are distributed to the opted in validators and their delegators.
+Note that rewards are **only** distributed to validators that are opted in and have been validating the consumer chain
+for a continuous number of epochs (see `NumberOfEpochsToStartReceivingRewards` param).
+
 To avoid spam, the provider must whitelist denoms before accepting them as ICS rewards.  
 
+## Reward distribution with power capping
```
Comment (insumity) [inline docs/docs/features/reward-distribution.md:25]: cc @ve6a

**Review by MSalopek — APPROVED** [review]

**Review by sainoe — APPROVED** [review]

**Pull request merged**